Project.toml

```
@@ -1,7 +1,7 @@
 name = "ONNXRunTime"
 uuid = "e034b28e-924e-41b2-b98f-d2bbeb830c6a"
 authors = ["Jan Weidner <jw3126@gmail.com> and contributors"]
-version = "0.3.3"
+version = "1.0.0"
```

```suggestion
version = "0.4"
```
See https://github.com/jw3126/ONNXRunTime.jl/pull/32#issuecomment-1763765490